packages/ketcher-macromolecules/src/components/macromoleculeProperties/MacromoleculePropertiesWindow.tsx

```diff
@@ -1057,14 +1057,9 @@ export const MacromoleculePropertiesWindow = () => {
       <GrossFormula data-testid="Gross-formula">
         {firstMacromoleculesProperties?.grossFormula
           .split(' ')
-          .map((atomNameWithAmount, i) => {
-            return (
-              <GrossFormulaPart
-                part={atomNameWithAmount}
-                key={i}
-              ></GrossFormulaPart>
-            );
-          })}
+          .map((atomNameWithAmount, i) => (
+            <GrossFormulaPart part={atomNameWithAmount} key={i} />
```

Comment:
It was `key={'${atomNameWithAmount}-${i}'}` in the previous version, now it reverted back to the index-based key